### PR TITLE
[EPO-3942] Fix any/all bugs found in audit

### PR DESCRIPTION
**Pull request by georgegomez144**
Add title to page and lang to html tag.
Update accessibility attributes in images and update color contrast in main navigation.
Update title tags to be consistent with heading levels rule
Add missing aria roles and aria labels

### CODE REVIEW

File: templates/teachers/guide.twig [hunk 1]
```diff
@@ -32,16 +32,16 @@
...
-            <li class="tabs-title"><a href="{{ teachersGuideUrl }}/phenomena"{{ _self.isActiveTab(currentTab, 'phenomena') }}>{{uielements.phenomenaText}}</a></li>
+            <li class="tabs-title{{ _self.isActiveTab(currentTab, 'phenomena') }}"><a href="{{ teachersGuideUrl }}/phenomena">{{uielements.phenomenaText}}</a></li>
         {% endif %}
         {% if assessments | length and isTeacher %}
-            <li class="tabs-title"><a href="{{ teachersGuideUrl }}/assessments"{{ _self.isActiveTab(currentTab, 'assessments') }}>{{uielements.assessmentsText}}</a></li>
+            <li class="tabs-title{{ _self.isActiveTab(currentTab, 'assessments') }}"><a href="{{ teachersGuideUrl }}/assessments">{{uielements.assessmentsText}}</a></li>
         {% endif %}
         {% if glossary | length %}
-            <li class="tabs-title"><a href="{{ teachersGuideUrl }}/glossary"{{ _self.isActiveTab(currentTab, 'glossary') }}>{{uielements.glossaryText}}</a></li>
+            <li class="tabs-title{{ _self.isActiveTab(currentTab, 'glossary') }}"><a href="{{ teachersGuideUrl }}/glossary">{{uielements.glossaryText}}</a></li>
```
Comment (blnkt) [inline templates/teachers/guide.twig:44]: role="tab" should be on all of the tabs, not just the active one.

File: templates/teachers/guide.twig [hunk 2]
```diff
@@ -32,16 +32,16 @@
       data-top-anchor="startTheStickyNavHere:top">
     <ul class="tabs c-teachers-guide-tabs" id="teachers-guide-tabs">
```
Comment (blnkt) [inline templates/teachers/guide.twig:33]: this ul should have role="tablist"

File: templates/_partials/investigation-grid.twig [hunk 3]
```diff
@@ -31,8 +31,9 @@
                 }).all() %}
 
                 <a class="c-card__link" href="{{ _self.getInvestigationLink(investigation, teacher, teacherGuide, isTeacher) }}">
-                    <img src="{{ investigation.heroImage.one().getUrl('small4x3') ?? uielements.hero.one().image.one().getUrl('small4x3') }}" />
-                    <h4>{{ investigation.title }}</h4>
+                    <img src="{{ investigation.heroImage.one().getUrl('small4x3') ?? uielements.hero.one().image.one().getUrl('small4x3') }}"
+                      alt="{{ investigation.title }} Investigation"/>
```
Comment (blnkt) [inline templates/_partials/investigation-grid.twig:35]: Don't need alt text.  Use role="presentation" instead.

File: templates/_partials/hero.twig [hunk 4]
```diff
@@ -2,7 +2,7 @@
 {% set hero = hero ?? defaultHero %}
 
 {% set heroImg = hero.image.one() ?? defaultHero.image.one() %}
-{% set headerText = hero.header ?? entry.title %}
+{% set headerText = hero.header ?? entry.title ?? headerText %}
```
Comment (blnkt) [inline templates/_partials/hero.twig:5]: What is this extra `?? headerText` doing?

**Review by blnkt — CHANGES_REQUESTED** [review]
Addressing the color contrast stuff in a different ticket?

> light green (#76923c) light orange(#f79646) and light blue (#548dd4) in the 3D lesson summary do not contrast well enough against their white background (please advise Ardis of how dark they would need to be, and/or add a custom palette to redactor, and/or, with Ardis, pick some other suitable colors that fit the bill); Alternatively, could maybe get away with changing default font size to 18px instead of 16px?

File: templates/_partials/hero.twig (hunk 4 above)
Comment (georgegomez144) [reply templates/_partials/hero.twig:5]: I broke this apart even further. This is a catch for no `entry.title`

File: templates/teachers/guide.twig (hunk 1 above)
Comment (georgegomez144) [reply templates/teachers/guide.twig:44]: Puttin it in the wrong area 👎 . This is updated.

File: templates/teachers/guide.twig (hunk 2 above)
Comment (georgegomez144) [reply templates/teachers/guide.twig:33]: added

File: templates/_partials/investigation-grid.twig (hunk 3 above)
Comment (georgegomez144) [reply templates/_partials/investigation-grid.twig:35]: 👍

**Comment by georgegomez144** [comment]
I have a note to update Ardis with 'acceptable' contrast colors after everything is pushed up. I tried updating the :root font size to 18px and it was just too "large" looking. It also messed with some fixed positioning that Foundation has put in place. Foundation is running its layout and sizing with `rem`s

File: templates/_partials/hero.twig (hunk 4 above)
Comment (blnkt) [reply templates/_partials/hero.twig]: But what does it do?  If there's no `hero.header`, and there's no `entry.title`, you are setting the value of `headerText` to `headerText`.  But how can be?  Aren't you instantiating that variable right here? Does this just result in an empty string like it is for headerContent?

**Review by blnkt — APPROVED** [review]

**Pull request merged**